package.json

```diff
@@ -16,7 +16,8 @@
     "coreUpgrade": "mvn test -Dcucumber.filter.tags='@selenium and @initialSetup and @upgrade'",
     "ocl": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @ocl'",
     "microfrontend": "./node_modules/.bin/cypress-tags run -eTAGS='@cypress and @microfrontend'",
-    "refApp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature"
+    "refApp3Login": "cypress run --spec src/test/resources/features/refapp-3.x/01-login/login.feature",
+    "vitalsAndTriagingTest": "mvn test -Dcucumber.filter.tags='@selenium and @vitals'"
```

Comment:
This configures the command that will trigger the test to run on every continuous integration builds.

Reply:
Sounds a good idea!